package.json

```diff
@@ -26,7 +26,12 @@
     "gulp-replace": "1.0.0",
     "gulp-sourcemaps": "2.6.5",
     "gulp-style-inject": "^0.1.3",
-    "gulp-watch": "5.0.1"
+    "gulp-stylelint": "^13.0.0",
+    "gulp-watch": "5.0.1",
+    "postcss": "^8.3.11",
+    "postcss-scss": "^4.0.1",
+    "stylelint": "^14.0.1",
+    "stylelint-config-standard-scss": "^2.0.0"
```

Comment:
Van jelentősége, hogy a gulp-watch pontos verziót követel?

Reply:
Ez literally az első commit óta így van :D. Nem hiszem, hogy különösebb jelentősége lenne, de az aktuális verzió amúgy [továbbra is ez](https://www.npmjs.com/package/gulp-watch).

Reply:
ok akkor